src/modules/core/producer_blank.yml

```diff
@@ -0,0 +1,12 @@
+schema_version: 0.3
+type: producer
+identifier: blank
+title: Blank
+version: 1
+copyright: Meltytech, LLC
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+  - Audio
+description: A blank frame generator.
```

Comment:
"blank" is too redundant in this documentation. What is the audio/video user experience of "blank" - black silence? Also, since this line is not a sentence it should not end with a period.

Reply:
I agree that the word is generic. But I am reusing the word as it is already defined in the framework:
https://github.com/mltframework/mlt/blob/b2787f9069004bb89af8d3d15d7761b024671197/src/framework/mlt_producer.c#L227

Here is a proposal for a more detailed description. Let me know what you think:

> A "Blank" producer returns frames that are transparent and silent. These frames are suitable for creating blank spaces in a playlist. mlt_producer_is_blank() will return 1 on this producer.

Reply:
"transparent" is not accurate because it is only considered transparent when the image format is rgba. Let's just say silent white. Otherwise, I like it's reference to `mlt_producer_is_blank`.

Reply:
done